app/scripts/background.js

```diff
@@ -801,6 +801,9 @@ export function setupController(
           case ApprovalType.SnapDialogConfirmation:
             controller.approvalController.accept(id, false);
             break;
+          case 'snap_manageAccounts:confirmation':
+            controller.approvalController.accept(id, false);
+            break;
```

Comment:
Why this one is not wrapped in the other fence?

```suggestion
          ///: END:ONLY_INCLUDE_IN
          ///: BEGIN:ONLY_INCLUDE_IN(keyring-snaps)
          case 'snap_manageAccounts:confirmation':
            controller.approvalController.accept(id, false);
            break;
```